docs/development/application-logic.md

```diff
@@ -237,9 +237,11 @@ transit processor (Littlepay).
 
```

Comment:
Line 235, `In this final phase, the user registers their contactless payment card with a concession group configured within the
transit processor (Littlepay).`, can be updated to include Switchio.

Reply:
How about just removing mention of the specific vendors from this sentence? Made that change in https://github.com/cal-itp/benefits/pull/2970/commits/510ec18cf2f417d985addf83bfed6635f2ad783e

Reply:
Yeah that sounds even better!